manta-js/CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Added
 - [\#51](https://github.com/Manta-Network/manta-signer/pull/51) Initial version of manta.js
+
```

Comment:
Add PR to changelog? Not sure we want two  separate ChangeLogs maybe just keep the top level one.

Reply:
yeah, will update

Reply:
Probably we just want to remove this changelog altogether.